.rubocop.yml

```diff
@@ -239,6 +239,11 @@ Style/RedundantRegexpEscape:
 Style/RedundantSelfAssignment:
   Enabled: true
 
+Style/RegexpLiteral:
+  Enabled: true
```

Comment:
It looks like we want `EnforcedStyle: mixed` here? That will enforce the use of `//` for single line literals and `%r{}` for multi line.